### PR TITLE
bump to go-ipfs v0.8.0-rc1

### PR DESCRIPTION
**Pull request by MichaelMure**

### CODE REVIEW

**Comment by MichaelMure** [comment]
@Stebalien I suspect I need to change the `ci-go` version but I don't have access to that repo. Could you give me a hand?

**Comment by Stebalien** [comment]
I have published 0.3.0 of the ci-go orb. Please update to that.

File: .circleci/config.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
   test-plugin:
     executor:
       name: ci-go/default
-      tag: 1.14.4
+      tag: 1.15.6
```
Comment (Stebalien) [inline .circleci/config.yml:19]: hm. this _should_ have worked.

**Comment by MichaelMure** [comment]
@Stebalien I updated ci-go, it's now broken in a new way :+1:

**Comment by Stebalien** [comment]
Ok... 0.3.1 should fix that.

**Comment by MichaelMure** [comment]
Back to square two :(

**Comment by MichaelMure** [comment]
@Stebalien a bit more help?

**Comment by Stebalien** [comment]
The aws sdk pulls in a newer version of sys. I'll make a PR against go-ipfs.

**Comment by Stebalien** [comment]
Well, I got it to work eventually. And surprisingly, I still hate go plugins!

**Comment by ianopolous** [comment]
Thank you!

**Pull request merged**